project/apps/question-service/package.json

```diff
@@ -21,8 +21,6 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@repo/pipes": "workspace:*",
-    "@repo/dtos": "workspace:*",
```

Comment:
i'm not very familiar with project dependencies, what happens if we remove these 2 lines?
will question-service still be able to reference dtos types?

Reply:
oh i removed it cause its a duplicate from line 30-31 haha